learn/advanced/known_limitations.md

```diff
@@ -88,3 +88,13 @@ user = 1 OR user = 2 […] OR user = 1500 OR user = 1501 […] OR user = 2000 OR
 **Limitation:** Meilisearch can only exactly represent integers between -2⁵³ and 2⁵³.
 
 **Explanation:** Meilisearch stores numeric values as double-precision floating-point numbers. This allows for greater precision and increases the range of magnitudes that Meilisearch can represent, but leads to inaccuracies in [values beyond certain thresholds](https://en.wikipedia.org/wiki/Double-precision_floating-point_format#Precision_limitations_on_integer_values).
+
+## Maximum number of results per search
+
+**Limitation:** Meilisearch returns 1000 documents per search.
+
+**Explanation:** This non-customizable limit ensures the database is protected from scraping. This limit only applies to the [search route](/reference/api/search.md). If you want to scrap your database, you can use the [get documents endpoint](/reference/api/documents.md#get-documents) instead.
+
+:::note
+This limit is different from the [`limit` search parameter](/reference/api/search.md#limit). Whereas `limit` restricts the number of results sent to the front-end, this means Meilisearch only has 1000 search results.
```

Comment:
I don't really understand what the second sentence is trying to say. Without too much context, I would suggest this:

```suggestion
This restriction is completely separate from the [`limit` search parameter](/reference/api/search.md#limit). Even if `limit` is greater than 1000, the search endpoint will only return 1000 documents.
```

Reply:
Rereading this and I agree it doesn't make sense, I'm not sure how to distinguish between the two "limits" here.
The first one is the [`limit` search parameter ](https://docs.meilisearch.com/reference/api/search.html#limit). It is customizable and restricts the number of results sent to the front-end
The other limit is the non-customizable 1000 maximum results per search.

So when I said:
>Whereas `limit` restricts the number of results sent to the front-end, this means Meilisearch only has 1000 search results. 

The "this" refers to the 2nd limit

Reply:
Hmm. Maybe I'm overthinking/misunderstanding, but I have the impression that we're talking about the distinction between an engine-level restriction and an API-level restriction.

`limit` affects the number of results sent to the front-end; it doesn't affect the number of results found by the search engine. So it's an API-level restriction.

The 1000 document hard limit affects the number of results found by the search engine: after it has found the 1000 best results, it will stop searching. So that's an engine-level restriction. At least that's my current understanding 🤷🏻 

This distinction is interesting, but not very helpful to most users in my opinion. The user just needs to understand that `limit` is not related to and does not affect the 1000 result limitation.

Reply:
Okay, I'll get rid of this note. We can add it later if needed